app/src/main/java/org/p2p/wallet/settings/ui/newreset/pin/NewForgotPinBottomSheet.kt

```diff
@@ -0,0 +1,48 @@
+package org.p2p.wallet.settings.ui.newreset.pin
+
+import androidx.core.os.bundleOf
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.setFragmentResult
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.p2p.wallet.R
+import org.p2p.wallet.common.ui.NonDraggableBottomSheetDialogFragment
+import org.p2p.wallet.databinding.DialogNewForgotPasswordBinding
+import org.p2p.wallet.utils.args
+import org.p2p.wallet.utils.viewbinding.viewBinding
+import org.p2p.wallet.utils.withArgs
+
+private const val EXTRA_REQUEST_KEY = "EXTRA_REQUEST_KEY"
+private const val EXTRA_RESULT_KEY = "EXTRA_RESULT_KEY"
+
+class NewForgotPinBottomSheet : NonDraggableBottomSheetDialogFragment() {
```

Comment:
it's better to add NonDraggableBottomSheetDialogFragment to proguard then

Reply:
Why?
Why should we add bottom sheet to proguard?

Reply:
i saw in sentry that proguard minifies fragment names for bottom sheets, gotta check it out, maybe i already added this rule

Reply:
I think it's expected behavior. 
We don't minify the class if it breaks the logic only. 
In other cases, we should be good